.github/workflows/updatecli.yml

```
@@ -19,11 +48,42 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        pipeline-name: [beats, golang, hermit]
+        pipeline-name: [ beats, golang, hermit ]
     steps:
       - uses: actions/checkout@v4
       - name: Init Hermit
         run: ./bin/hermit env -r >> $GITHUB_ENV
+      - name: Set Branch
+        run: echo "GIT_BRANCH=main" >> $GITHUB_ENV
+      - uses: elastic/apm-pipeline-library/.github/actions/updatecli@current
+        with:
+          vaultUrl: ${{ secrets.VAULT_ADDR }}
+          vaultRoleId: ${{ secrets.VAULT_ROLE_ID }}
+          vaultSecretId: ${{ secrets.VAULT_SECRET_ID }}
+          pipeline: ./.ci/updatecli/updatecli.d/update-${{ matrix.pipeline-name }}.yml
+          values: ./.ci/updatecli/values.yml
+          notifyIfFailure: false
+
+  updatecli-backport:
+    name: Update ${{ matrix.pipeline-name }} dependencies - backport
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        pipeline-name: [ golang, beat ]
```

nit: for consistency 
```suggestion
        pipeline-name: [ golang, beats ]
```